src/FsAutoComplete/FsAutoComplete.Lsp.fs

```diff
@@ -760,48 +761,46 @@ type FsharpLspServer(commands: Commands, lspClient: FSharpLspClient) =
                 {ci with Detail = Some name; Documentation = Some d  }
             | CoreResponse.Res (HelpText.Full (name, tip, additionalEdit)) ->
                 let (si, comment) = (TipFormatter.formatTip tip) |> List.collect id |> List.head
-                //TODO: Add insert namespace
+                let edits, label =
```

Comment:
this is a new fix/feature that adds the 'open <blah>' text to the file like the TODO said.

Reply:
Wait, so it was broken since we moved Ionide to LSP?

Reply:
I don't know about that far back, but I think we have multiple places where open expressions can be suggested and this one in particular was broken.

I know I _have_ seen open namespaces for things like the quick fix when you use a type from a namespace that's not opened working as intended.

Reply:
Yes, we should (and used to in pre-LSP days) suggest it in 2 places:
* As a quick fix
* As a result of autocomplete (inserting open statements) - this is (or should be) disabled by default (because it can pollute autocomplete results quite a lot)

Reply:
so I guess with this change the second point would be considered 'on by default'? I'm personally ok with it because it's clear from the tooltip when an open will be performed (thanks to the `(open <namespace>)` text I added as part of this.

Reply:
Are you ok with this change in behavior?

Reply:
Well, whether those suggestions are on or off is still controlled here - https://github.com/fsharp/FsAutoComplete/pull/671/files/e41cca0e608a87a66c9b2fff25b1edcdb6975d24#diff-9c5c2a3ad10bb0f644010469df24332399c9273b1e420e8ef48b44d64f49c402R723

Reply:
And `config.ExternalAutocomplete` is false by default in Ionide

Reply:
OOOOHHHHHHHH

Ok I see. Then we still have the same old configuration behavior (which is fine) I just always have that config flag on personally :)